### PR TITLE
Install hipfft for ROCm 4.2

### PR DESCRIPTION
**Pull request by takagi**
Following https://github.com/cupy/cupy/pull/5318.

### CODE REVIEW

**Comment by kmaehashi** [comment]
Thanks!!
* I think it can be `system_packages` (instead of `rocm_packages`) so that ROCm-specific code can be removed from Dockerfiles.
* Could you fix flake8 errors?

**Comment by takagi** [comment]
Okay, I'm going to follow the points!

**Review by kmaehashi — APPROVED** [review]

File: verifier/Dockerfile.debian [hunk 1]
```diff
@@ -41,12 +41,17 @@ RUN for VERSION in ${python_versions}; do \
       pip install numpy scipy; \
     done
 
-# Install additional dependencies for ROCm.
+# Tentatively pin verifier to use rocm-4.0
 RUN [ ! -d /opt/rocm ] || ( \
-        export DEBIAN_FRONTEND=noninteractive && \
         sed -ie 's|http://repo.radeon.com/rocm/apt/debian/|http://repo.radeon.com/rocm/apt/4.0/|' /etc/apt/sources.list.d/rocm.list && \
+    )
```
Comment (takagi) [inline verifier/Dockerfile.debian:47]: Is this only for rocm-4.0?

**Comment by takagi** [comment]
PTAL!

**Comment by kmaehashi** [comment]
Thanks!!
/test

**Comment by leofang** [comment]
Silly question: why https://github.com/cupy/cupy-release-tools/commit/af7fdbf5e0a564d739d29433e3a925aa9098a1fa didn't work?

**Comment by kmaehashi** [comment]
@leofang The base image (`[rocm/dev-centos-7:4.0.1](https://hub.docker.com/r/rocm/dev-centos-7)`) is not using multi-version packages, so it conflicts if we use multi-versioned packages😢

File: verifier/Dockerfile.debian [hunk 2]
```diff
@@ -41,12 +41,17 @@ RUN for VERSION in ${python_versions}; do \
       pip install numpy scipy; \
     done
 
-# Install additional dependencies for ROCm.
-RUN [ ! -d /opt/rocm ] || ( \
-        export DEBIAN_FRONTEND=noninteractive && \
+# Tentatively pin verifier to use rocm-4.0
+RUN [ ! -d /opt/rocm-4.0.0 ] || ( \
         sed -ie 's|http://repo.radeon.com/rocm/apt/debian/|http://repo.radeon.com/rocm/apt/4.0/|' /etc/apt/sources.list.d/rocm.list && \
```
Comment (kmaehashi) [inline verifier/Dockerfile.debian:46]:
```suggestion
        sed -ie 's|http://repo.radeon.com/rocm/apt/debian/|http://repo.radeon.com/rocm/apt/4.0/|' /etc/apt/sources.list.d/rocm.list \
```

**Review by kmaehashi — CHANGES_REQUESTED** [review]

**Comment by kmaehashi** [comment]
Locally confirmed build succeed :tada:

**Comment by kmaehashi** [comment]
Will backport after https://github.com/cupy/cupy-release-tools/pull/121

**Pull request merged**